make/photon/prepare/utils/nginx.py

```diff
@@ -17,12 +17,17 @@
 CUSTOM_NGINX_LOCATION_FILE_PATTERN_HTTP = 'harbor.http.*.conf'
 
 def prepare_nginx(config_dict):
-    prepare_config_dir(nginx_confd_dir)
+    file_path = prepare_config_dir(nginx_confd_dir)
+    os.chown(file_path, DEFAULT_UID, DEFAULT_GID)
     render_nginx_template(config_dict)
 
 def render_nginx_template(config_dict):
     if config_dict['protocol'] == "https":
-        render_jinja(nginx_https_conf_template, nginx_conf,
+        render_jinja(
```

Comment:
does nginx conf file need `uid` `gid`?

Reply:
of course

Reply:
I see they are parms to the func that mark the output file.